frontend/locale/de.json

```diff
@@ -17,65 +17,65 @@
+  "list.footer.clear_completed": "Fertige Einträge löschen",
   "my_lists.no_lists": "Keine Listen vorhanden",
   "my_lists.new_list": "Erstelle eine neue Liste!",
   "new_list.header": "Erstelle eine neue Liste",
   "new_list.placeholder": "Geben Sie einen Listennamen ein.",
-  "new_list.button": "Erstelle eine neue Liste!",
-  "news.browse_recipe_button": "Durchsuchen Sie alle Rezepte",
+  "new_list.button": "Meine Liste erstellen!",
+  "news.browse_recipe_button": "Alle Rezepte durchsuchen",
   "autocomplete.loading": "Laden...",
   "autocomplete.no_matches": "Keine Übereinstimmung mit {value} gefunden.",
   "directions.next_direction": "Nächste Anweisung",
   "ingredients.add_ingredient": "Zutat hinzufügen",
-  "subrecipe.add_link": "Add Link",
+  "subrecipe.add_link": "Link hinzufügen",
   "directions.step_label": "Anweisung {step, number}",
   "directions.textarea_placeholder": "Anweisung",
   "ingredients.name_placeholder": "Bezeichnung",
   "ingredients.quantity_placeholder": "Menge",
   "ingredients.measurement_placeholder": "Einheit",
-  "ingredients.group_placeholder": "Group (leave blank for default)",
+  "ingredients.group_placeholder": "Gruppe (leer lassen für Standardeinstellung)",
   "subrecipe.name_placeholder": "Name",
-  "subrecipe.quantity_placeholder": "Quantity",
-  "subrecipe.measurement_placeholder": "Measurement",
-  "recipe.import.header": "Import A Recipe",
-  "recipe.import.source_label": "Enter a URL to gather recipe data",
-  "recipe.import.websites": "Supported Websites",
+  "subrecipe.quantity_placeholder": "Menge",
+  "subrecipe.measurement_placeholder": "Einheit",
+  "recipe.import.header": "Ein Rezept importieren",
+  "recipe.import.source_label": "Eine URL eingeben um Rezeptdaten zu erfassen",
+  "recipe.import.websites": "Unterstützte Webseiten",
   "recipe.import.import": "Import",
-  "recipe.import.importing": "Importing",
-  "recipe.import.error": "Error",
-  "recipe.import.error_message_1": "Something went wrong! Please try again.",
-  "recipe.import.error_message_2": "This Website is not supported.",
-  "recipe.import.error_message_3": "Please enter a valid URL.",
-  "recipe.import.hide_sites": "hide supported sites",
-  "recipe.import.show_sites": "show supported sites",
+  "recipe.import.importing": "Importiere",
+  "recipe.import.error": "Fehler",
+  "recipe.import.error_message_1": "Etwas ging schief! Bitte versuchen Sie es noch einmal.",
+  "recipe.import.error_message_2": "Diese Webseite wird nicht unterstützt.",
+  "recipe.import.error_message_3": "Bitte eine zulässige URL eingeben.",
+  "recipe.import.hide_sites": "unterstützte Seiten verbergen",
+  "recipe.import.show_sites": "unterstützte Seiten anzeigen",
   "recipe.create.name_label": "Rezepttitel",
   "recipe.create.name_placeholder": "Rezept",
   "recipe.create.course_label": "Gang",
   "recipe.create.cuisine_label": "Küche",
-  "recipe.create.tags_label": "Tags",
+  "recipe.create.tags_label": "Schlagworte",
```

Comment:
i translated "recipe.create.tags_label" to match "searchbar.placeholder".
although "tag" is also used as anglicism in german language, some tech-unsavvy german native speakers have no idea what "tags" are.